networking-cisco/rpm/build-rpm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Should be run from the root of the source tree
+
+if [ ! -d rpm ]; then
+   echo "Directory 'rpm' not found"
+   exit 1
+fi
+SPEC_FILE_IN=`ls rpm/*.spec.in`
+if [ -z $SPEC_FILE_IN ]; then
+   echo "RPM spec file not found"
+   exit 1
+fi
+BUILD_DIR=${BUILD_DIR:-`pwd`/rpmbuild}
+mkdir -p $BUILD_DIR/BUILD $BUILD_DIR/SOURCES $BUILD_DIR/SPECS $BUILD_DIR/RPMS $BUILD_DIR/SRPMS
+NAME=`python setup.py --name`
+RELEASE=${RELEASE:-1}
+VERSION_PY=`python setup.py --version`
+#VERSION=${VERSION_PY/.dev/~dev}
```

Comment:
Because of the way setup.py reports "dev" versions, it is not directly useable with RPMs. Can you explain why this line is not needed? BTW, please remove code that is no longer used instead of commenting it out.